rho/scancommand.py

```diff
@@ -399,15 +399,21 @@ def _validate_options(self):
 
         # perform fact validation
         facts = self.options.facts
-        if facts == [] or facts == ['default']:
-            self.facts_to_collect = list(utilities.DEFAULT_FACTS_TUPLE)
+        if facts == [] or facts == ['default'] or facts == ['all']:
+            self.facts_to_collect = list(utilities.DEFAULT_FACTS)
+        elif facts == ['jboss']:
+            self.facts_to_collect = list(utilities.JBOSS_FACTS +
```

Comment:
What is the difference between a fact group that has the `_TUPLE` suffix and the ones that does not have it?

Reply:
Thanks